airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -298,6 +301,31 @@ public void create(final Map<String, String> allLabels,
     final List<ContainerPort> containerPorts = KubePodProcess.createContainerPortList(portMap);
     containerPorts.add(new ContainerPort(serverPort, null, null, null, null));
 
+    final var initContainer = new ContainerBuilder()
+        .withName(KubePodProcess.INIT_CONTAINER_NAME)
+        .withImage("busybox:1.35")
+        .withVolumeMounts(volumeMounts)
+        .withCommand(List.of(
+            "sh",
+            "-c",
+            String.format("""
+                          i=0
+                          until [ $i -gt 60 ]
+                          do
+                            echo "$i - waiting for config file transfer to complete..."
+                            if [ -f "%s/%s" ]; then
+                              exit 0
+                            fi
+                            i=$((i+1))
+                            sleep 1
+                          done
+                          echo "config files did not transfer in time"
```

Comment:
I think we should already have some logic on the launcher side to check if the orchestrator took more than a certain time to start.
Should opt for some "crazy high" value (10~30min??) here just as a failsafe and rely on the other logic to deal with slow starting orchestrators.

Reply:
This sort of [already exists](https://github.com/airbytehq/airbyte/blob/9ab7f46c5dcb0f8738898a74ceaf483ebc8f0756/airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java#L365-L367) as the launcher will wait for up to 5 minutes before ultimately bailing out (due to the [status not matching](https://github.com/airbytehq/airbyte/blob/9ab7f46c5dcb0f8738898a74ceaf483ebc8f0756/airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java#L380-L382))

Reply:
So my take is that since we already have the launcher wait up to 5min, I would not have a timeout in the init-container, the upload can be slow as long as the orchestrator starts within the time. Somehow, I think that setting the init-container to fail earlier adds more configuration to check a similar outcome, so I'd remove it.

To be clear, I am not feeling super strongly either way about this.

Reply:
The timeout in the init-container matches the prior (still existing as of this PR) timeout in the container-orchestrator where that contain would kill itself if the file wasn't present within 60s of starting up.

I'm not sure where that original 60s setting comes from, and I'm open to changing it.

Reply:
To clarify; the 5m wait is only waiting on the container to be available, the 60s is how long we're giving for the files to copy over.

Reply:
My take would be to have simpler conditions, I am not sure we are getting a whole lot of values from all those different timeouts. Those settings might be easier to consolidate once we switched to micronaut since it would set us up with a clearer configuration pattern.